contributors/devel/cri-validation.md

```diff
@@ -0,0 +1,50 @@
+# CRI Validation Testing
+
+CRI validation testing provides a series of common validation test suites for Kubelet CRI (_Container Runtime Interface_).
+
+It helps container runtime maintainers to validate their runtime when integrating with kubelet CRI.
+
+*Note: No kubelet is running. The test suites run directly targeting the shim of container runtime.*
+
+*Note: CRI validation testing is alpha and maintained at [cri-tools](https://github.com/kubernetes-incubator/cri-tools) repo.*
+
+## Install
+
+The test suites could be installed easily via `go get` command:
+
+```sh
+go get github.com/kubernetes-incubator/cri-tools/cmd/critest
+```
+
+Then `critest` binary could be found in `$GOPATH/bin`.
+
+*Note: ensure GO is installed and GOPATH is set before installing critest.*
+
+## Running tests
+
+Before running the tests, ensure the shim of CRI container runtime is running and kubelet is stopped.
```

Comment:
Suggestion:
"
*Prerequisite*

Before running the test, you need to ensure that the CRI server under test is running and listening on a Unix
socket. Because the validation tests are designed to request changes (e.g., create/delete) to the containers and verify that correct status is reported, it expects to be the only user of the CRI server. Please make sure that 1) there are no existing CRI-managed containers running on the node, and 2) no other processes (e.g., Kubelet) will interfere with the tests.
"